app/common/DocumentSettings.ts

```diff
@@ -2,6 +2,7 @@ export interface DocumentSettings {
   locale: string;
   currency?: string;
   engine?: EngineCode;
+  idOfDefaultAttachmentStore?: string;
```

Comment:
My motivation for phrasing it this way specifically was the ambiguity of the variable if it ends in `id`.

It could be:
`defaultAttachmentStore Id`
or
`default attachmentStoreId`

Which are semantically different interpretations of the same name! I'd be happy to swap it around if you think `defaultAttachmentStoreId` is clear enough though, despite the above :)

Reply:
`attachmentStoreId` is my favorite now I'm afraid